pyrep/objects/shape.py

```diff
@@ -209,12 +210,22 @@ def set_mass(self, mass: float) -> None:
         sim.simSetObjectFloatParameter(
             self._handle, sim.sim_shapefloatparam_mass, mass)
 
-    def get_mesh_data(self) -> Tuple[List[float], List[float], List[float]]:
+    def get_mesh_data(self, asnumpy=None):
```

Comment:
Let's just change this to always returning a tuple of np arrays (i.e. remove `asnumpy`). Can you add the return types please? 
`Tuple[np.ndarray, np.ndarray, np.ndarray]`